session.rst

```diff
@@ -1,15 +1,276 @@
+.. index::
+   single: HTTP
+   single: HttpFoundation, Sessions
+
 Sessions
 ========
 
-Symfony provides a session object and several utilities that you can use to
-store information about the user between requests.
+Installation
+------------
```

Comment:
```suggestion
```

(just a suggestion, I never like articles starting with a sub-title)

Reply:
Refactored to avoid that.